_includes/payment-resource.md

```diff
@@ -141,6 +156,23 @@ for the given operation.
 
 {% else %}
 
+{% case showStatusOperations %}
```

Comment:
Are you using `case` to have `else` available? Is `else` not possible to use in conjunction with a regular `if`?